src/connectors/assetpanda.py

```diff
@@ -0,0 +1,290 @@
+                parse_number(  entry.get('Status', None)  ),
+                parse_number(  entry.get('Purchase_date', None)  ),
+                parse_number(  entry.get('Yubikey_Identifier', None)  ),
+                parse_number(  entry.get('display_name', None)  ),
+                parse_number(  entry.get('Brand', None)  ),
+                parse_number(  entry.get('Assigned_To', None)  ),
+                parse_number(  entry.get('share_url', None)  ),
+                parse_number(  entry.get('object_version_ids', None)  ),
+                parse_number(  entry.get('Creation_Date', None)  ),
+                parse_number(  entry.get('Created_By', None)  ),
+                parse_number(  entry.get('purchase_price', None)  ),
+                parse_number(  entry.get('next_service', None)  ),
+                parse_number(  entry.get('building', None)  ),
+                parse_number(  entry.get('category', None)  ),
+                parse_number(  entry.get('description', None)  ),
+                parse_number(  entry.get('changed_by', None)  ),
+                parse_number(  entry.get('wireless_status', None)  ),
+                parse_number(  entry.get('created_at', None)  ),
+                parse_number(  entry.get('gps_coordinates', None)  ),
+                parse_number(  entry.get('updated_at', None)  ),
+                parse_number(  entry.get('loaner_pool', None)  ),
+                parse_number(  entry.get('default_attachment', None)  ),
+                parse_number(  entry.get('room', None)  ),
+                parse_number(  entry.get('notes', None)  ),
+                parse_number(  entry.get('object_depreciation', None)  ),
+                parse_number(  entry.get('is_editable', None)  ),
+                parse_number(  entry.get('wifi_mac_address', None)  ),
+                parse_number(  entry.get('change_date', None)  ),
+                parse_number(  entry.get('display_size', None)  ),
+                parse_number(  entry.get('operating_system', None)  ),
+                parse_number(  entry.get('serial', None)  ),
+                parse_number(  entry.get('end_of_life_date', None)  ),
+                parse_number(  entry.get('imei_meid', None)  ),
+                parse_number(  entry.get('model', None)  ),
+                parse_number(  entry.get('mac_address', None)  ),
+                parse_number(  entry.get('entity', None)  ),
+                parse_number(  entry.get('PO', None)  )
+            ) for entry in list_object_without_field_id],
+            select=db.derive_insert_select(LANDING_TABLE_COLUMNS),
+            columns=db.derive_insert_columns(LANDING_TABLE_COLUMNS)
+        )
+        
+        log.info(f'Inserted {len(list_object_without_field_id)} rows ({landing_table}).')
+        yield len(list_object_without_field_id)
+
+        # increment the offset to get new entries each iteration in the while loop
+        params["offset"] += PAGE_SIZE
```

Comment:
```suggestion
        params["offset"] += PAGE_SIZE
        
```

Reply:
What's the difference?

Reply:
You added a blank line at the end?

Reply:
files should end with a newline

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline